packages/engine/Source/Scene/I3SField.js

```diff
@@ -92,40 +101,37 @@ function getNumericTypeSize(type) {
   return 0;
 }
 
+function getValueTypeSize(type) {
+  if (type === "String") {
+    return 1;
+  }
+  return getNumericTypeSize(type);
+}
+
+async function load(field) {
+  const data = await field._dataProvider._loadBinary(field._resource);
+  const dataView = new DataView(data);
+  field._data = data;
+  field._validateHeader(dataView);
+  const headerSize = field._parseHeader(dataView);
+  const offset = field._getBodyOffset(headerSize);
+  field._validateBody(dataView, offset);
+  field._parseBody(dataView, offset);
+}
+
 /**
  * Loads the content.
  * @returns {Promise<void>} A promise that is resolved when the field data is loaded
  */
 I3SField.prototype.load = function () {
-  const that = this;
-  return this._dataProvider._loadBinary(this._resource).then(function (data) {
-    // Check if we have a 404
-    const dataView = new DataView(data);
-    let success = true;
-    if (dataView.getUint8(0) === "{".charCodeAt(0)) {
-      const textContent = new TextDecoder();
-      const str = textContent.decode(data);
-      if (str.includes("404")) {
-        success = false;
-        console.error(`Failed to load: ${that.resource.url}`);
-      }
-    }
-
-    if (success) {
-      that._data = data;
-      let offset = that._parseHeader(dataView);
-
-      const valueSize = getNumericTypeSize(
-        that._storageInfo.attributeValues.valueType
-      );
-      if (valueSize > 0) {
-        // Values will be padded to align the addresses with the data size
-        offset = Math.ceil(offset / valueSize) * valueSize;
-      }
+  if (defined(this._loadPromise)) {
+    return this._loadPromise;
+  }
 
-      that._parseBody(dataView, offset);
-    }
+  this._loadPromise = load(this).catch(function (error) {
+    console.error(error);
```

Comment:
Ideally, the error should be thrown so that the developer can handle it in their app accordingly. 

Can we remove the catch and console.log?

Reply:
For this one, we recommend keeping the catch. This logic handles any errors with availability or binaries for attributes. When such a condition is present (the issue is related to source I3S data not having complete information and/or unexpected buffer sizes) app developers can do nothing to fix it. Catching these errors allows working with valid attributes, considering invalid attributes as missing. 
See below screenshot; the door asset would still be able to show the attributes associated with it with the error catching and logs the issue; without error catching, this asset wouldn’t be able to display the attribute pop up at all.
![image](https://github.com/CesiumGS/cesium/assets/12946567/b338611e-ad3b-4e66-aaaf-d6d7a0e6b07d)